rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/IndexJavaSourceFilesRuleProvider.java

```diff
@@ -76,7 +80,7 @@ private IndexJavaFileIterationOperator()
         @Override
         public void perform(GraphRewrite event, EvaluationContext context, JavaSourceFileModel payload)
         {
-            WindupRoasterWildcardImportResolver.setGraphContext(event.getGraphContext());
+            WindupWildcardImportResolver.setGraphContext(event.getGraphContext());
```

Comment:
It's lame, but it's also not a new problem with this PR (we've been using this approach for a while). I'd love to have a nicer solution.